### ISSUE TITLE
Registers with long names are tructated with elipsis

### ISSUE THREAD

**Issue by hydra** [marcin-bukowiecki/regdebug#4]
there is no way to display the full name.

![image](https://user-images.githubusercontent.com/57075/136375387-0e91c2d3-901b-4392-ab08-773e8c78cd07.png)

**Comment by marcin-bukowiecki** [comment]
Yes, this is possible by executing the `set` command through GDBDriver. I will try to add this in the next release.
